src/core/TabPanel/TabPanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { useTheme } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: any;
+  value: any;
+}
+
+const TabPanel: React.FC<TabPanelProps> = ({
```

Comment:
Tab Panel is already provided by MUI as a `lab` component, why do we need to create a separate `Tab Panel` component and import it in `Tab` from here? Is there any requirement for this? Like any customizations?

Reply:
@S-ayanide Initially, TabPanel and Tab have been a common component. For the rest check the comment above please

Reply:
@S-ayanide 
These two modules don't exist, as we are using MUI core, thus we can't extend from these modules:
import TabPanel from '@material-ui/lab/TabPanel';
// or
import { TabPanel } from '@material-ui/lab';

THat's why TabPanel component function is given manually.

Reply:
TabPanel is present as a lab component in MUI, core is for Kuubera-UI thus we could have used a lab component from MUI to the core of kubera-ui.

But we anyways won't require to have a `TabPanel` thus there is no requirement to create it as a separate local component,

Reply:
@S-ayanide If we don't need to have the TabPanel, then where to show each Tab's content?

Reply:
We can pass content directly as a child, so we can keep it at the main `Tab.tsx`